engine/runner/src/main/scala/org/enso/runner/Main.scala

```diff
@@ -1056,9 +1056,13 @@ object Main {
         exitFail()
     }
 
-  /** Default log level to use if the LOG_LEVEL option is not provided.
+  /** Default log level to use when starting a language server if the
+    * `LOG_LEVEL` option is not provided.
     */
-  private val defaultLogLevel: Level = Level.WARN
+  private val defaultServerLogLevel: Level = Level.WARN
```

Comment:
Can we have/update documentation describing:
- what to do to log a message into console?
- what to do to log a message into log file?
- what to do to log a stack trace into log file?
- what to do to log a stack trace into console?
- how to do that with sl4j?
- how to do that with `TruffleLogger`?

Reply:
> what to do to log a message into console?
> what to do to log a message into log file?

I don't see how those are related. Besides one should not care at this point where we are logging. 

> how to do that with sl4j?
> how to do that with TruffleLogger?

Isn't this kind of self-explanatory? The former - use slf4j Logger, the latter - use TruffleLogger.

> what to do to log a stack trace into log file?
> what to do to log a stack trace into console?

OK, that one "might" deserve some clarification though it's not related to this PR.
Since **all** layout is controlled in a single configuration it's a question of adding `%ex` vs `%noex` in the pattern field, eg. https://github.com/enso-org/enso/blob/develop/engine/launcher/src/main/resources/application.conf#L30.

Reply:
> one should not care at this point where we are logging

Isn't the whole point of this issue that "unexpected warning" gets logged? If we had clear answers to 

> what to do to log a message into console?
> what to do to log a message into log file?

we would know what `Level` to use to suppress the message.

> ... it can be debug

No, it cannot be debug. There is no debug in `java.util.logging.Level`... 

> how to do that with sl4j?
> how to do that with TruffleLogger?

That's why I am asking for examples of code to achieve each action in both systems.